internal/cephfs/cephfs_util.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"fmt"
 
+	cerrors "github.com/ceph/ceph-csi/internal/cephfs/util/errors"
```

Comment:
Just using `"github.com/ceph/ceph-csi/internal/cephfs/errors"` would be fine. No need to have a `util` package under `cephfs` I think?

Reply:
Done